server/index.js

```diff
@@ -21,6 +21,7 @@ app.use(bodyParser.urlencoded({extended: false}));
 app.use('/auth', authRouter);
 app.use('/todoList', todoListRouter);
 app.use('/share', shareRouter);
+app.use('/category', todoListRouter);
```

Comment:
ok, so category will be new path but still routed to todoListRouter... because it will be sending same calls as /todoList. gotcha.